### ISSUE TITLE
Ingest MR data

### ISSUE THREAD

**Issue by wrgr** [openconnectome/ocp-journal-paper#55]
@jovo 

Not sure where this is on the priority stack for @gkiar, but was the other issue open from the big issues of issues.

**Comment by jovo** [comment]
i think after @gkiar does a successful knn run on the m2g data, this would be the top priority.
@gkiar does that sound right? if so, please add appropriate milestone.

**Comment by wrgr** [comment]
added milestone.

**Comment by gkiar** [comment]
Define successful? :P I agree, though.

## 

Greg Kiar gkiar07@gmail.com

**Comment by jovo** [comment]
success = 1 DTI or fMRI scan ingested, link to viz (either using RESTful call or ndviz) posted in this issue.

**Comment by wrgr** [comment]
@jovo - clarify priority here please?  

Are we allowed to use console in the short term?

**Comment by jovo** [comment]
we _always_  _always_  _always_  _always_ do the simplest thing first.
if that's console, then yes.

**Comment by wrgr** [comment]
clarify priority here please?  i understood your requirements were not to use the console.  this makes an initial version achievable quickly.  

@gkiar - put a sample data set of your choice somewhere?

**Comment by jovo** [comment]
requirement: one MR dataset ingest _any possible way you want_.

note: this is not a @willgray activity, it is a @gkiar activity.
@willgray is very busy with case studies and does not have time for this
until case studies are _completely_ finished.

On Wed, Jun 15, 2016 at 11:15 PM, William Gray notifications@github.com
wrote:

> clarify priority here please? i understood your requirements were not to
> use the console. this makes an initial version achievable quickly.
> 
> @gkiar https://github.com/gkiar - put a sample data set of your choice
> somewhere?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/neurodata/ndpaper/issues/55#issuecomment-226378048,
> or mute the thread
> https://github.com/notifications/unsubscribe/AACjciDojEi0ZgzJxTx5Nl1i3iBnl65Pks5qML-5gaJpZM4Iykne
> .

## 

the glass is all full:  half water, half air.
neurodata.io, jovo calendar
https://calendar.google.com/calendar/embed?src=joshuav%40gmail.com&ctz=America/New_York

**Comment by wrgr** [comment]
thanks.  I'll aim for ~end of july if not solved prior.

**Comment by wrgr** [comment]
@gkiar - this is done?

**Comment by gkiar** [comment]
Yes, actually the entire KKI2009 dti dataset, with the token "KKI2009dti"

**Comment by wrgr** [comment]
if you don't mind - send me the info and we'll stick in the table for the ndpaper?

I need the stuff for Table 1 (image data).  I can be more specific or draft if you don't have time.

**Comment by wrgr** [comment]
```
Reference Paper: 
Modality:  MR
Species:  H. sapiens
Bits: 
# Projects:  1
# Channels:  42
# Time Steps: 1
# Gigavoxels:
# Resolutions: 1
# Gigabytes:
```

@alexbaden 

Double check the above and please fill out missing info.  Note GV = GB \* bit depth and these are measured in GB, so for MR an approximate guess is ok.

http://openconnecto.me/ocp/ca/KKI2009dti/info/

**Comment by alexbaden** [comment]
I'm going to have to script it. Give me 36 hours.

**Comment by alexbaden** [comment]
```
Reference Paper: 
Modality:  MR
Species:  H. sapiens
Bits: 8
# Projects:  1
# Channels:  42
# Time Steps: 1
# Gigavoxels: 62.23
# Resolutions: 1
# Gigabytes: 57.96
```

That what you need?

@gkiar if you can spot check this at some point, that would be great! No rush.

**Comment by wrgr** [comment]
reference paper is:  Bennett. A. Landman, Alan J. Huang, Aliya Gifford, Deepti S. Vikram, Issel Anne L. Lim, Jonathan A.D. Farrell, John A. Bogovic, Jun Hua, Min Chen, Samson Jarso, Seth A. Smith, Suresh Joel, Susumu Mori, James J. Pekar, Peter B. Barker, Jerry L. Prince, and Peter C.M. van Zijl. “Multi-Parametric Neuroimaging Reproducibility: A 3T Resource Study”, NeuroImage. (2010) NIHMS/PMC:252138 doi:10.1016/j.neuroimage.2010.11.047

@gkiar - once you are happy with this (spot check), close and I'll add to the ndpaper

**Comment by wrgr** [comment]
Oh - and thanks - @alexbaden  -  you beat your estimate by a factor of ~3x!
